wd_helper.au3

```diff
@@ -636,6 +636,199 @@ Func _WD_FrameLeave($sSession)
 	Return SetError(__WD_Error($sFuncName, $_WD_ERROR_Success), 0, $sValue)
 EndFunc   ;==>_WD_FrameLeave
 
+; #FUNCTION# ====================================================================================================================
+; Name ..........: _WD_FrameList
+; Description ...: get list of documents (window.top and all frames) from current webdriver context/tab
```

Comment:
A few comments about this --

- If it returns a list of documents, then I think the function should be renamed to something that better describes the action being performed
- It isn't clear to me what you mean by "from current webdriver context/tab". Does this function return a listing of all frames for the current tab or only those from the current browsing context?

Reply:
> * If it returns a list of documents, then I think the function should be renamed to something that better describes the action being performed

_WD_GetFrames ?

Reply:
No. For me, the issue is that it seems like you are using "documents" in the description as if it were interchangeable with "frames", when they are two different things.

Reply:
so maybe like this:
`; Description ...: get list of frames (window.top and all frames) from current webdriver context`

Reply:
I know that `window.top` is not frame but WebDriver treat it like frames when it comes to Frame Entering as using NULL you are leaving all frames and you are on top but you must use "FRAME" function to do this.

Reply:
better description proposal:
`; Description ...: list detailed information about frames possible to use (window.top and all frames)`

Reply:
suplemented in:
[further changes](https://github.com/Danp2/au3WebDriver/pull/362/commits/3bdf5437e1f1792e68f9d69804f593ce4cd6ac52)

Reply:
>  ; Description ...: list detailed information about frames possible to use ("window.top" and all frames) 

Some people may not be familiar with "window.top". Here's my take on how this should read --

` ; Description ...: Retrieves a detailed list of the main document and all associated frames`

Reply:
Please take a look here: [const parameters + header](https://github.com/Danp2/au3WebDriver/pull/362/commits/6276ff5011d2a1cc604dbffceb19079f7100ac47)